### PR TITLE
Fix typos in PySMO import statements (documentation)

### LINKED ISSUE #401: Typing error in PySMO
Hello,

I found a typo in the following link example:

https://lbl-idaes-dev.readthedocs-hosted.com/en/1.5.1/surrogate/pysmo/pysmo_sampling_properties.html#sampling-details

When you are importing the module, you import as **idaes.surrogates** instead of **idaes.surrogate**.

Took me some time to check if I had the my env setup correctly and was actually a problem with conda. It is a easy fix so I thought I should mention it ! 
Keep up the good work ! :)

### PR DESCRIPTION
**Pull request by OOAmusat**
## Fixes
#401 

## Summary/Motivation:
The name of the file containing the surrogate tools was changed from "surrogates" to "surrogate" a while back - this PR seeks to correct the docs to reflect that.

## Changes proposed in this PR:
- change `idaes.surrogates.pysmo` to ` idaes.surrogate.pysmo` in docs

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by lbianchi-lbl — APPROVED** [review]
I searched for "`idaes.surrogates`" in the codebase, and all of them seem to addressed by this PR, so this looks good to me.

The pylint check is failing because of #407, which is unrelated to this PR. We can either wait until #408 is merged in and subsequently merge back those changes into this PR, or just ignore the failing check (since pylint only looks at Python files, and would not catch any potential error in this PR).

**Review by jghouse88 — APPROVED** [review]
LGTM

**Pull request merged**
